content/logs/aws.md

```diff
@@ -0,0 +1,133 @@
+---
+title: AWS Log Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+## Overview
+
+This documentation explains how to create a Lambda function in AWS to forward S3 logs to Datadog.
```

Comment:
Let's change this as we do not only target S3 logs:

AWS offers all machines, all services, all you need to develop, run and operate your product or app.
Last decade, Amazon developed and added many services. But it's still complicated to troubleshoot and understand what's going on beside an issue or a major outage.

Reply:
👍 :